dummy_lib/core.py

```diff
@@ -3,33 +3,53 @@
 # This program is licensed under the Apache License 2.0.
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0> for full license details.
 
+from enum import Enum
 from typing import List
 
-__all__ = ["greet_contributor", "convert_fahrenheit_to_celsius"]
+__all__ = ["greet_contributor", "convert_temperature_sequences", "TemperatureScale"]
+
+
+class TemperatureScale(Enum):
+    """Temparature scale
```

Comment:
[Low priority]
There seems to be a typo here, would you mind correcting it?

```suggestion
    """Temperature scale
```

This feedback is about: [understandability] [docstring]. If this is wrong, click [here](https://github.com/frgfm/ci-benchmark/pull/url).